build.sbt

```diff
@@ -138,7 +138,8 @@ val sharedSettings = List(
           "-Ywarn-unused:imports"
         )
     }
-  }
+  },
+  Test / scalacOptions := (Compile / scalacOptions).value
```

Comment:
Can we also add a comment pointing to https://github.com/sbt/sbt/issues/5934, so that it's clear why we need this?